hawkbit-azure-extensions-parent/pom.xml

```diff
@@ -0,0 +1,44 @@
+<!--
```

Comment:
I'd prefer the root named e.g. `hawkbit-extension-azure` or `hawkbit-extensions-azure` so it is more consistent with the other extensions